src/libsyncengine/jobs/network/API_v2/upload/upload_session/abstractuploadsession.cpp

```diff
@@ -35,50 +35,50 @@
 namespace KDC {
 
 AbstractUploadSession::AbstractUploadSession(const SyncPath &filepath, const SyncName &filename,
-                                             uint64_t nbParalleleThread /*= 1*/) :
-    _logger(Log::instance()->getLogger()), _filePath(filepath), _filename(filename), _nbParalleleThread(nbParalleleThread) {
-    IoError ioError = IoError::Success;
+                                             const uint64_t nbParallelThread /*= 1*/) :
+    _logger(Log::instance()->getLogger()), _filePath(filepath), _filename(filename), _nbParallelThread(nbParallelThread) {
+    auto ioError = IoError::Success;
     if (!IoHelper::getFileSize(_filePath, _filesize, ioError)) {
-        std::wstring exceptionMessage = L"Error in IoHelper::getFileSize for " + Utility::formatIoError(_filePath, ioError);
+        const std::wstring exceptionMessage = L"Error in IoHelper::getFileSize for " + Utility::formatIoError(_filePath, ioError);
         LOGW_WARN(_logger, exceptionMessage.c_str());
         throw std::runtime_error(Utility::ws2s(exceptionMessage).c_str());
     }
 
     if (ioError == IoError::NoSuchFileOrDirectory) {
-        std::wstring exceptionMessage = L"File doesn't exist: " + Utility::formatSyncPath(_filePath);
+        const std::wstring exceptionMessage = L"File doesn't exist: " + Utility::formatSyncPath(_filePath);
```

Comment:
```suggestion
        const std::wstring exceptionMessage = L"File does not exist: " + Utility::formatSyncPath(_filePath);
```
Minor and arguable: easier to read; reserve quotes for quoting paths and names.